src/controllers/controlleroutputmappingtablemodel.cpp

```diff
@@ -189,6 +190,53 @@ QVariant ControllerOutputMappingTableModel::data(const QModelIndex& index,
     return QVariant();
 }
 
+QString ControllerOutputMappingTableModel::displayString(const QModelIndex& index) const {
```

Comment:
It is not clear from the function name if the function is an imperative or a getter. 
Can we improve it? Something like `getStringForSearch()`

Alternative we may integrate the code into data() itself using a new user role 
e.g. `SearchRole = Qt::UserRole + 1`

Reply:
True, let's call it `getDisplayString` then?

Tbh I'd rather stick to explicit `QString getDisplayString` instead of having to deal with even more type conversion and more safety checks in `data().`

Reply:
OK

Reply:
Thanks for the idea to use a custom role!